media/js/base/core-datalayer-init.js

```diff
@@ -4,14 +4,31 @@
 
 // init core dataLayer object and push into dataLayer
 $(function() {
-    if (Mozilla && Mozilla.Analytics) {
-        var dataLayer = window.dataLayer = window.dataLayer || [];
+    var dataLayer = window.dataLayer = window.dataLayer || [];
```

Comment:
Do we really need to repeat this seeing as it's already defined as a global in the [GTM snippet](https://github.com/mozilla/bedrock/blob/master/bedrock/base/templates/includes/google-analytics.html#L7)? If we do need it for some reason, could it be shortened to `var dataLayer = window.dataLayer || [];`?